src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.html

```diff
@@ -54,8 +54,8 @@
         </form>
     </div>
 
-    <div class="secondary-toolbar-left"></div>
-    <div class="secondary-toolbar-right">
+    <div class="lower-toolbar-left"></div>
+    <div class="lower-toolbar-right">
```

Comment:
This approach are not conceptually I mean, on the design side, we don't necessarily have to have the "secondary" bar "below", if we decided to change the design and now have the content of the "lower" bar in the top the work "lower" will make no sense.

In HTML and CSS we can't name things by their look and feel because if that looks and feel change the name with not match.